web/templates/stream_settings/announce_stream_checkbox.hbs

```diff
@@ -0,0 +1,15 @@
+<label class="checkbox">
+    <input type="checkbox" name="announce" value="announce" checked />
+    <span></span>
+    {{t "Announce new stream in"}}
+    <strong>
+        {{#if notification_stream_sub}}
+            {{> ../stream_privacy
```

Comment:
Can we use `inline_decorated_stream_name` here? Currently there is some space between the name and icon and we are intending to avoid that space in other existing UIs as well.

Reply:
Yes, can be used. I have updated the PR.